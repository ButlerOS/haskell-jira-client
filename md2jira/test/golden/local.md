---
assigned: user@corp
---

This is my backlog.

# Develop app

## Implement X

The goal is...
