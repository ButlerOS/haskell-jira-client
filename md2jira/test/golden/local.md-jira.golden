EpicTitle: Develop app
EpicBody:


StoryTitle: Implement X
StoryBody:
The goal is...


